lib/customs.js

```diff
@@ -76,16 +76,13 @@ module.exports = function (log, error) {
       }
     )
     .then(
-      function (result) {
-        return { lockout: !!result.lockout }
-      },
+      function () {},
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
         // If this happens, either:
         // - (1) the url in config doesn't point to a real customs server
         // - (2) the customs server returned an internal server error
         // Either way, allow the request through so we fail open.
-        return { lockout: false }
```

Comment:
I'm fine with us returning undefined here and it'll mean less churn for the tests, however, having an empty function with no explanation feels a little untidy.  Maybe worth a comment along the lines of:

```
// There's no useful information in the HTTP response,
// so we just discard it
```

To indicate that we're doing this on purpose?